pom.xml

```diff
@@ -288,16 +291,74 @@
 	</dependencies>
 
 	<distributionManagement>
-		<repository>
-			<id>deployment</id>
-			<name>Internal Releases</name>
-			<url>https://nexus.afrigis.co.za/content/repositories/releases/</url>
-		</repository>
 		<snapshotRepository>
-			<id>snapshots-deployment</id>
-			<name>Internal Releases</name>
-			<url>https://nexus.afrigis.co.za/content/repositories/snapshots/</url>
+			<id>ossrh</id>
+			<url>https://oss.sonatype.org/content/repositories/snapshots</url>
 		</snapshotRepository>
+		<repository>
+			<id>ossrh</id>
+			<url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+		</repository>
 	</distributionManagement>
+
 	<description>Service API Implementation for AfriGIS Services</description>
+
```

Comment:
Entire section based on [the guide.](http://central.sonatype.org/pages/apache-maven.html)